.gitignore

```diff
@@ -0,0 +1,4 @@
+.Rproj.user
```

Comment:
For the `.gitignore`, would it make sense to also not track any additional .txt files that get output by some plotting functions (i.e `onco_matrix.txt` from `prettyOncoplot`). There might be additional items we would not like to track on Git. But maybe this can be addressed at a later stage.